screens/overviewMap.js

```diff
@@ -40,20 +40,25 @@ const styles = StyleSheet.create({
     position: 'absolute',
     bottom: 20,
     margin: 10,
-    paddingHorizontal: 6,
-    paddingVertical: 3,
     borderRadius: 100,
     borderWidth: 1,
-    borderColor: 'rgba(3, 3, 3, 0.4)',
-    backgroundColor: 'rgba(228, 229, 227, 1)',
+    borderColor: colours.black,
+    backgroundColor: colours.white,
   },
   buttonIcon: {
-    backgroundColor: 'rgba(0,0,0,0)',
+    backgroundColor: colours.transparent,
   },
   addButton: {
+    paddingTop: 14,
```

Comment:
These are all different because the icons aren't perfectly sized 😕

Reply:
I think because they are all different, one wrapper probably wouldn't work... I'll make a comment